app/models/main.py

```diff
@@ -1832,6 +1848,302 @@ def serialize(self):
         }
 
 
+class ProcessOutcome(db.Model):
+    __tablename__ = "process_outcomes"
+
+    RESULT_CHOICES = (
+        "awarded",
+        "cancelled",
+        "none-suitable",
+    )
+
+    id = db.Column(db.Integer, primary_key=True)
+    external_id = db.Column(db.BigInteger, nullable=False, default=random_positive_external_id, unique=True)
+    completed_at = db.Column(db.DateTime, nullable=True)
+
+    start_date = db.Column(db.Date, nullable=True)
+    end_date = db.Column(db.Date, nullable=True)
+    awarding_organisation_name = db.Column(db.String, nullable=True)
+    # should be enough scale to represent up to £1b - 1p
+    award_value = db.Column(db.Numeric(11, 2), nullable=True)
+
+    result = db.Column(db.String, nullable=False)
+
+    direct_award_project_id = db.Column(
+        db.Integer,
+        db.ForeignKey('direct_award_projects.id'),
+        nullable=True,
+    )
+    direct_award_search_id = db.Column(
+        db.Integer,
+        db.ForeignKey('direct_award_searches.id'),
+        nullable=True,
+    )
+    direct_award_archived_service_id = db.Column(
+        db.Integer,
+        db.ForeignKey('archived_services.id'),
+        nullable=True,
+    )
+
+    brief_id = db.Column(
+        db.Integer,
+        db.ForeignKey('briefs.id'),
+        nullable=True,
+    )
+    brief_response_id = db.Column(
+        db.Integer,
+        db.ForeignKey('brief_responses.id'),
+        nullable=True,
+    )
+
+    # a concise wrapper to reduce the ugliness of the workaround for https://bitbucket.org/zzzeek/sqlalchemy/issues/3999
+    # (which should no longer be needed once we switch to sqlalchemy 1.2)
```

Comment:
We should get this onto the tech debt board

Reply:
:+1:

Reply:
Because if you look at the number of times it is used in the following code, it bloats the expressions out to something horrible and scary. That's *why* it's only one character, because the lines overflow enough as it is. And syntactically it's supposed to be transparent (should be readable as though it's not there at all). I'd rather it *weren't* there at all - I *wanted* to use `_` but george didn't like that as it's normally used for translations.

Reply:
No great need to argue about this seeing as it ought to go away, given some (fairly high priority I think) piece of tech debt getting done. I don't like the idea of us falling way being on ORM versions (as and when we are doing significant dev on the project anyway, erm.)